src/genie/libs/parser/asa/show_access_list.py

```diff
@@ -0,0 +1,309 @@
+''' show_access_list.py
+
+Parser for the following show commands:
+    * show access-list
+'''
+
+# Python
+import re
+import logging
+from collections import OrderedDict
```

Comment:
No need to use an OrderedDict, please use a regular dictionary

Reply:
In access lists it makes a difference in which order the entries appear, for example when looping over them it is not the same if we use a regular dict or an OrderedDict. An OrderedDict will provide them in the right order and a regular dict will not guarantee it.